scons/site_tools/arm.py

```diff
@@ -208,6 +208,7 @@ def generate(env, **kw):
 		"-Wl,-wrap,_calloc_r",
 		"-Wl,-wrap,_realloc_r",
 		"-Wl,-wrap,_free_r",
+		"--specs=nosys.specs",
 		"--specs=nano.specs",
```

Comment:
what does gcc do with two specs files?
is one ignored or are they merged somehow?